pyof/v0x01/symmetric/echo_request.py

```diff
@@ -30,4 +30,4 @@ def __init__(self, xid=None, data=None):
             data (bytes): arbitrary-length data field.
         """
         super().__init__(xid)
-        self.data = data
+        self.data = BinaryData() if data is None else data
```

Comment:
`BinaryData()` -> `b''`; it's bad having GenericTypes when they are not needed.
The cleaner way is to set `b''` directly in the `__init__` method - same behavior, better readability.